src/App.js

```diff
@@ -37,21 +40,31 @@ function App() {
     return (
     <div className="App main-container">
         <Navbar></Navbar> 
-        <CartCard/>
-        <Footer/>
+        {/* <CartCard/> */}
+        {/* <CartCard/> */}
         {/* <div className="main-content">
         <HeroImg/>
         </div> */}
         {/* <div className="categories-card">
         <CategoryCard/>
         </div> */}
```

Comment:
Good thing that you have a vision of what the next step is, but while merging branches don't keep any commented code. if you want to experiment then make a sub-branch from the current branch and try in that branch.